css/styles.css

```diff
@@ -1,715 +1,1549 @@
+  display: flex;
+  justify-content: center;
+}
+
+.pictures__image {
+  width: 147px;
+}
+
+.pictures__image:last-child {
+  display: none;
+}
+
+@media screen and (max-width: 1024px) {
+  .section-stories {
+    margin: 0 auto 94px;
+  }
+
+  .section-stories-top {
+    margin: 0 auto 87px;
+  }
+
+  .stories-content {
+    width: 276px;
+  }
+
+  .stories-content__title {
+    margin: 26px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 13px;
+    line-height: 16px;
+    width: 277px;
+    margin: 0 0 22px 0;
+  }
+
+  .stories-content__photo {
+    width: 277px;
+    height: 149px;
+  }
+
+  .more__link {
+    width: 110px;
+  }
+
+  .pictures__image {
+    width: 133px;
+    height: 135px;
+  }
+
+  .pictures__image_display-none {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 940px) {
+  .section-stories-top {
+    justify-content: space-around;
+  }
+
+  .stories-content:last-child {
+    display: none;
+  }
+
+  .pictures__image:nth-child(6),
+  .pictures__image:nth-child(7) {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .section-stories {
+    background-color: #000;
+    color: #fff;
+    padding: 63px 0 0 0;
+    position: relative;
+    margin: 0 auto 353px;
+  }
+
+  .section-stories__title {
+    font-size: 30px;
+  }
+
+  .section-stories-top {
+    justify-content: flex-start;
+    padding: 0 0 0 43px;
+  }
+
+  .stories-content {
+    width: 298px;
+    margin: 0 71px 0 0;
+  }
+
+  .stories-content__title {
+    font-size: 25px;
+    margin: 26px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 20px;
+    line-height: 25px;
+    width: 312px;
+    margin: 0 0 37px 0;
+  }
+
+  .stories-content__photo {
+    width: 312px;
+    height: 165px;
+  }
+
+  .more__link {
+    font-size: 20px;
+    width: 204px;
+    color: #fff;
+  }
+
+  .more__link::after {
+    background: url("../images/logo/arrow-w.svg") right no-repeat;
+    background-size: cover;
+    width: 49px;
+    height: 16px;
+    top: 5px;
+  }
+
+  .more__data {
+    font-size: 15px;
+    margin: 4px -33px 0 0;
+  }
+
+  .stories-content:nth-child(2) 
+  .more__data {
+    margin: 5px -14px 0 0;
+  }
+
+  .section-stories-bottom {
+    height: 193px;
+  }
+
+  .section-stories::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    width: 100%;
+    height: 36px;
+    background: linear-gradient(to bottom left, #000 50%, #fff 52%);
+  }
+
+  .stories-bottom__hashtag {
+    font-size: 30px;
+    padding: 37px 43px 0 0;
+    margin-bottom: 10px;
+  }
+
+  .pictures {
+    max-width: 680px;
+    margin: 0 auto;
+    flex-wrap: wrap;
+    font-size: 0;
+    position: relative;
+    z-index: 1;
+  }
+
+  .pictures__image {
+    width: 170px;
+    height: 171px;
+  }
+
+  .pictures__image:nth-child(6),
+  .pictures__image:nth-child(7),
+  .pictures__image:last-child {
+    display: inline;
+  }
+}
+
+@media screen and (max-width: 745px) {
+  .section-stories {
+    margin-bottom: 410px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Itogovya_axion/pullRequest?prid=6118030)